libs/utils/android/workloads/camerarecord.yaml

```diff
@@ -0,0 +1,7 @@
+config:
+    device: generic_android
+workloads:
+    - id: b01
```

Comment:
Maybe use a more meaningful name?
I guess this is reported in logs as well... something like "lisa_camerarecord_20" ?